lib/list.js

```diff
@@ -41,23 +47,27 @@ function validate (opts) {
   if (opts.age && !R.contains(opts.age, R.values(c.age))) {
     throw Error(`Invalid age range ${opts.age}`);
   }
+}
+
+function defineCollectionUrl (opts) {
```

Comment:
instead of adding this sort of logic, can you update the object in constants to already contain the path that needs to be called?